src/p2p/net_peerlist_boost_serialization.h

```diff
@@ -95,7 +95,9 @@ namespace boost
     {
       uint32_t ip{na.ip()};
       uint16_t port{na.port()};
+      ip = SWAP32LE(ip);
       a & ip;
+      ip = SWAP32LE(ip);
```

Comment:
port?

Reply:
Port is saved below. This line is to restore ip after it was temporarily swapped.

Reply:
Gotcha